src/content/plugins/split-chunks-plugin.md

```diff
@@ -270,7 +270,6 @@ module.exports = {
   //...
   optimization: {
     splitChunks: {
-      automaticNamePrefix: 'general-prefix',
```

Comment:
Please move the example to `cacheGroups`.

Reply:
there is one, check line 277

Reply:
We need to move the whole thing including the heading, heading should now become smth like
```
#### `splitChunks.cacheGroups.{cacheGroup}.automaticNamePrefix`
```